package.json

```diff
@@ -58,7 +58,7 @@
       }
     },
     "start": {
-      "command": "babel-node bin/server",
+      "command": "npm run lint && babel-node bin/server",
```

Comment:
Hey @michael-martin, it would be good to get your thoughts on my response. Also, I can't tell if @vitkon is agreeing with that thumbs up or not, but would be good to have your opinion as well.

This is only one part of it but I want to get consensus across DDUK FED about how we structure our projects.